docs/docs/general/locale.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Locale
+---
+
+For applications where the numbers of ticks on scales must be formatted accordingly with a language sensitive number formatting, you can enable this kind of formatting setting `locale` option.
```

Comment:
Isn't: you can enable this kind of formatting setting with the `locale` option a better english sentence?

Reply:
I have changed following @LeeLenaleee hint (2nd option)